package.json

```diff
@@ -14,7 +14,9 @@
     "framer-motion": "^10.16.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.16.0"
+    "react-router-dom": "^6.16.0",
+    "react-to-pdf": "^1.0.1",
+    "react-to-print": "^2.14.13"
```

Comment:
Is this dependency also used?

Reply:
It isn't used, it already got removed in my last push